combinations/hash.tsv

```diff
@@ -192,3 +192,4 @@ bioconductor-biocparallel=1.28.0,bioconductor-csaw=1.28.0,bioconductor-edger=3.3
 r-base=4.1.2,r-alakazam=1.2.0,r-shazam=1.1.0,r-kableextra=1.3.4,r-knitr=1.33,r-stringr=1.4.0,r-dplyr=1.0.6
 python=3.9.5,samtools=1.14
 bioconductor-diffbind=3.4.0,bioconductor-csaw=1.28.0,bioconductor-edger=3.36.0,r-optparse=1.7.1
+r-base=4.1.2,r-shazam=1.1.0,r-tigger=1.0.0
```

Comment:
you do not need to specify r-base, its a dependency of r-shazam

Reply:
ok, thanks!